config/src/main/java/org/springframework/security/config/annotation/web/configurers/UrlAuthorizationConfigurer.java

```diff
@@ -49,10 +49,11 @@
  * </p>
  *
  * <pre>
- * protected void configure(HttpSecurity http) throws Exception {
- * 	http.apply(new UrlAuthorizationConfigurer&lt;HttpSecurity&gt;()).getRegistry()
- * 			.antMatchers(&quot;/users**&quot;, &quot;/sessions/**&quot;).hasRole(&quot;USER&quot;)
- * 			.antMatchers(&quot;/signup&quot;).hasRole(&quot;ANONYMOUS&quot;).anyRequest().hasRole(&quot;USER&quot;);
+ * @Bean
+ * public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
```

Comment:
The method has to receive the `ApplicationContext` as a parameter, like this:

```suggestion
 * public SecurityFilterChain filterChain(HttpSecurity http, ApplicationContext context) throws Exception {
```